MapboxMobileEventsTests/MMEEventsManagerTests.mm

```diff
@@ -788,10 +784,7 @@ - (void)pushEvent:(MMEEvent *)event;
         
         beforeEach(^{
             dateString = @"A nice date";
-            spy_on(dateWrapper);
-            dateWrapper stub_method(@selector(formattedDateStringForDate:)).and_return(dateString);
-            eventsManager.dateWrapper = dateWrapper;
-            
+
```

Comment:
```suggestion
            NSDateFormatter *dateFormatter = MMEDate.iso8601DateFormatter;
            spy_on(dateFormatter);
            dateFormatter stub_method(@selector(stringFromDate:)).and_return(dateString);
            
```

Reply:
(never used these suggested changes -- so, feel free to do what you want with that)